# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: media/webrtc/signaling/src/media-conduit/AudioConduit.h
# Commit: 909f472c2e40
# Full Hash: 909f472c2e402f81a8326e93e839666d2aba1fc5
# Author: Dan Minor <dminor@mozilla.com>
# Date: 2018-11-03 09:44:53
# Regressor Bug: 1376873
# File Overlap Count: 2
# Description:
#   Bug 1376873 - Add RtpPacketQueue; r=pehrsons
#   
#   We'll need to queue rtp packets in both the AudioConduit and the VideoConduit.
#   This adds a class to manage the packet queue to reduce duplicated code between
#   the conduits.
# ==============================================================================

diff -r 19d9a30b300c -r 909f472c2e40 media/webrtc/signaling/src/media-conduit/AudioConduit.h
--- a/media/webrtc/signaling/src/media-conduit/AudioConduit.h	Tue Feb 20 15:23:09 2018 -0500
+++ b/media/webrtc/signaling/src/media-conduit/AudioConduit.h	Thu Mar 22 09:44:49 2018 -0400
@@ -170,6 +170,10 @@
 
   virtual uint64_t CodecPluginID() override { return 0; }
   virtual void SetPCHandle(const std::string& aPCHandle) override {}
+  MediaConduitErrorCode DeliverPacket(const void *data, int len) override
+  {
+    return kMediaConduitNoError;
+  }
 
   virtual void DeleteStreams() override {}
 